ci/pipelines/deploy-to-staging.yml

```diff
@@ -1123,7 +1123,7 @@ jobs:
       - load_var: application_image_tag
         file: egress-ecr-registry-staging/tag
       - task: create-notification-snippets
-        file: pay-ci/ci/tasks/create-notification-snippets.yml
+        file: pay-ci/ci/tasks/create-egress-notification-snippets.yml
```

Comment:
I don't think the snippet accepts the `ACTION_NAME` var yet, that'll need updating too.

Reply:
What the.... fixed.